core/example/solvers/definitions.h

```diff
@@ -0,0 +1,67 @@
+#ifndef TDS_DEFINITIONS_INCLUDED
+#define TDS_DEFINITIONS_INCLUDED
+
+#include <Cabana_AoSoA.hpp>
+#include <Cabana_MemberTypes.hpp>
+
+#include <Kokkos_Core.hpp>
+
+
+#define PI 3.141592653589793238462643
+#define PI_SQRT 1.772453850905516
+#define PI_SQ 9.869604401089359
+#define PI_DIV_SQ 0.101321183642338
+
+#define COULOMB_PREFACTOR 1.0
+#define COULOMB_PREFACTOR_INV 1.0
+
+#define MADELUNG_NACL -1.747564594633182
```

Comment:
Prefer `constexpr` variables to preprocessor macros.

Reply:
Switched these to `constexpr`